src/components/PersonMetadata/PersonMetadata.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react'
```

Comment:
This is perhaps a minor thing but why are wecalling it "PersonMetadata" it's not really just metadata it's about presenting, why not just call it Person?

This perhaps is a larger conversation of how we should organise components perhaps.

Reply:
To comment to myself, I actually see whats going on here a bit more now, it's to seperate out from amongst other data that might display on an details page.